### DISCUSSION TITLE
Playlist of certain files parts

### DISCUSSION THREAD

**Discussion by fehC** [smplayer-dev/smplayer#438] [Q&A]
Hello,

Is it possible to create a playlist of parts of files?
Example:

item1:
file: <filepath>
time-start-end: 00:05:22.151-00:15:10.078
time-start-end: 00:20:01.039-01:18:43.221

item2:
file: <filepath>
etc...

Thanks

**Comment by smplayer-dev** [comment]
It's not supported, but I guess you can create [edl files](https://github.com/mpv-player/mpv/blob/master/DOCS/edl-mpv.rst).
